doc/intro.rst

```diff
@@ -99,24 +99,26 @@ To have it globally persistent add to your NixOS configuration file py3status as
 .. code-block:: nix
 
     (python3Packages.py3status.overrideAttrs (oldAttrs: {
-      propagatedBuildInputs = [ python3Packages.pytz python3Packages.tzlocal ];
+      propagatedBuildInputs = with python3Packages;[ pytz tzlocal ] ++ oldAttrs.propagatedBuildInputs;
```

Comment:
We could use other packages as example here, since `pytz` and `tzlocal` is already included in the NixOS derivation.